uiowa/uiowa
# Moving to `project/classy` in preparation for D10

Resolves https://github.com/uiowa/uiowa/issues/6277
Resolves https://github.com/uiowa/uiowa/issues/5131

This PR gives us updated classy styles that haven't been updated in over 2 years. 

# How to test
Locally inspect markup and look for `` instead of ``

```
ddev composer install && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /form/terms-of-service

```
Confirm terms of service pop-up is now working.
